### ISSUE TITLE
crash occurs when latitude axis includes the poles

### ISSUE THREAD

**Issue by brian-rose** [brian-rose/climlab#94]
Here is a minimal example of setting up a
```
import numpy as np
import climlab
num_lat = 4
lat = np.linspace(-90., 90., num_lat)
state = climlab.column_state(lat=lat)
rad = climlab.radiation.RRTMG(state=state)
```

Trying to take a step forward with `rad.step_forward()` results in a segmentation fault (no error message).

**Comment by brian-rose** [comment]
The crash is presumably related to the nonsensical assumptions that are made about grid cell boundaries:
```
print(rad.lat)
print(rad.lat_bounds)
```
produces
```
[-90. -30.  30.  90.]
[-90. -60.   0.  60.  90.]
```

i.e. the outer boundaries are equal to the cell centers of the first and last grid points.

There should be an error raised somewhere before this is allowed to crash.

**Comment by brian-rose** [comment]
Turns out this problem was fixed by #98 

Nothing to do with grid boundaries (even though they are nonsensical) but rather the ozone interpolation returning NaNs.

Now a warning is issued and extrapolated values are used.
